lib/__tests__/resolveConfig.test.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const path = require('path');
+const pluginWarnAboutFoo = require('./fixtures/plugin-warn-about-foo');
+const replaceBackslashes = require('../testUtils/replaceBackslashes');
+const stylelint = require('..');
+
+describe('resolveConfig', () => {
+	it('resolve to undefined without a path', () => {
```

Comment:
[trivial] Looks a typo:

```suggestion
	it('should resolve to undefined without a path', () => {
```

or

```suggestion
	it('resolves to undefined without a path', () => {
```

Reply:
Done in 3d2ec8b.